osfs/os.go

```diff
@@ -168,6 +168,42 @@ func (fs *OS) RemoveAll(path string) error {
 	return os.RemoveAll(fullpath)
 }
 
+// Symlink creates newname as a symbolic link to oldname.
+// All parent directories are created.
+func (fs *OS) Symlink(oldname, newname string) error {
```

Comment:
Indeed, that's a compelling argument for the name of a function (`Symlink` instead of `SymLink`) as it minimises surprise and increases familiarity with the API and that is why I didn't mention it in my review.

On the other hand, I think it's a weak argument for the function arguments (`oldname` instead of `oldName`) as none of the above is true for them and in some cases, it can be mistaken for cut-and-paste laziness.